hgext/postfix_hook.py

```diff
@@ -0,0 +1,35 @@
+# This software may be used and distributed according to the terms of the
+# GNU General Public License version 2 or any later version.
+
+import json
+
+from mercurial import (
+    pycompat,
+)
+
+testedwith = b"5.5"
+
+# For clarity
+PASS, FAIL = 0, 1
+
+def postfixfile_hook(ui, repo, rev=None, path=None, metadata=None, 
+                     **kwargs):
+    """Hook that runs after each file changed in `hg fix`."""
+    pass
+
+
+def postfix_hook(ui, repo, replacements=None, wdirwritten=False, 
+                 **kwargs):
+    """Hook that runs after `hg fix` is complete."""
+    if wdirwritten:
+        ui.warn(b"Working directory was written; this should not happen\n")
+        return FAIL
+
+    #if replacements:
+    #    for k, v in replacements.items():
+    #        ui.write(b"key=%s, val=%s\n" % 
+    #                 (pycompat.bytestr(k.decode('utf-8', 'surrogatepass')), 
+    #                  pycompat.bytestr(v.decode('utf-8', 'surrogatepass'))))
+
```

Comment:
My plan is to put information about re-written commits here and parse them from the output of `run_hg(["fix...`